CMakeLists.txt

```diff
@@ -133,23 +133,23 @@ option(INCLUDE_LINUX_I2C "Include I2C support" YES)
 IF (INCLUDE_LINUX_I2C)
 	CHECK_INCLUDE_FILES ("sys/types.h;linux/i2c-dev.h;linux/i2c.h" HAVE_LINUX_I2C_H)
 	IF (HAVE_LINUX_I2C_H)
-		message(STATUS "Building with I2C support")
+		message(SEND_ERROR "Building with I2C support")
```

Comment:
Please change this back to status, you can see what this does in our build system (it errors out... but it should not)